src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -242,10 +243,10 @@ public void TestEnvironmentOfChildProcess()
             }
         }
 
-        [PlatformSpecific(TestPlatforms.Windows)] // UseShellExecute currently not supported on Windows on .NET Core
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Sorry, that's what I meant. `[SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)]` should skip Unix, right?